### PR TITLE
(FEV-1028): Player v7|Navigation 2.1.5| Accessibility does not work properly

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

fix accessibility and add tooltip on clear button
Solves FEV-1028

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/navigation-search/navigation-search.tsx [hunk 1]
```diff
@@ -107,11 +109,20 @@ export class NavigationSearch extends Component<SearchProps, SearchState> {
...
+              label="Search in Video"
+              type="bottom"
+              className={styles.tooltip}>
+              <Fragment>
+                <button
+                  className={styles.clearIcon}
+                  onClick={this._onClear}
+                  tabIndex={1}
+                />
+              </Fragment>
```
Comment (RoyBregman) [inline src/components/navigation-search/navigation-search.tsx:123]: What does a Fragment help here? I think it is redundant...

File: src/components/navigation/navigation-item/NavigationItem.tsx [hunk 2]
```diff
@@ -160,6 +160,7 @@ export class NavigationItem extends Component<Props, State> {
     } = data;
     return (
       <div
+        tabIndex={1}
```
Comment (RoyBregman) [inline src/components/navigation/navigation-item/NavigationItem.tsx:163]: I hate using tabIndex different than -1 and 0. Is this necessary? Can't we use 0?

File: src/components/navigation-search/navigation-search.tsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
-import {h, Component} from 'preact';
+import {h, Component, Fragment} from 'preact';
```
Comment (RoyBregman) [inline src/components/navigation-search/navigation-search.tsx:1]: Remove Fragment

File: src/components/navigation-search/navigation-search.tsx (hunk 1 above)
Comment (JonathanTGold) [reply src/components/navigation-search/navigation-search.tsx:124]: Unfortunately not

File: src/components/navigation-search/navigation-search.tsx [hunk 4]
```diff
@@ -107,11 +109,21 @@ export class NavigationSearch extends Component<SearchProps, SearchState> {
...
+            <Tooltip
+              label="Search in Video"
+              type="bottom"
+              className={styles.tooltip}>
+              {/* Fragment added to overcome css issues */}
+              <Fragment>
+                <button
+                  className={styles.clearIcon}
+                  onClick={this._onClear}
+                  tabIndex={1}
```
Comment (RoyBregman) [inline src/components/navigation-search/navigation-search.tsx:122]: This can also be 0?

Comment (JonathanTGold) [reply src/components/navigation-search/navigation-search.tsx]: for you everything is possible

**Review by RoyBregman — APPROVED** [review]

**Pull request merged**